### PR TITLE
Feature NT-194 Dialogue presenter

### PR DESCRIPTION
**Pull request by PatrickSlag**

### CODE REVIEW

**Review by VarioCode — DISMISSED** [review]

File: Assets/Scripts/Presenters/DialoguePresenter.cs [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    public class DialoguePresenter : MonoBehaviour
+    {
+        [SerializeField] private GameObject dialogueBox;
+        [SerializeField] private TMP_Text dialogueText;
+        [SerializeField] private Button nextDialogueButton;
+        [SerializeField] private Button DebugDialogueButton;
+
+        private Queue<IMessage> _dialogueQue = new Queue<IMessage>();
+
+        public void QueMessages(IMessage[] messages)
```
Comment (jortdr) [inline Assets/Scripts/Presenters/DialoguePresenter.cs]: Queue

File: Assets/Scripts/Presenters/DialoguePresenter.cs [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+namespace Presenters
+{
+    public class DialoguePresenter : MonoBehaviour
+    {
+        [SerializeField] private GameObject dialogueBox;
+        [SerializeField] private TMP_Text dialogueText;
+        [SerializeField] private Button nextDialogueButton;
+        [SerializeField] private Button DebugDialogueButton;
+
+        private Queue<IMessage> _dialogueQue = new Queue<IMessage>();
```
Comment (jortdr) [inline Assets/Scripts/Presenters/DialoguePresenter.cs]: Queue

**Review by jortdr — APPROVED** [review]

**Review by jortdr — CHANGES_REQUESTED** [review]
Que?

**Review by VarioCode — DISMISSED** [review]
No more que nice

**Review by VarioCode — DISMISSED** [review]

**Review by jortdr — DISMISSED** [review]
Que

**Review by VarioCode — DISMISSED** [review]

**Review by jortdr — DISMISSED** [review]

**Review by jortdr — APPROVED** [review]

**Review by VarioCode — APPROVED** [review]

**Pull request merged**